Framework/DataHandling/test/SaveFocusedXYETest.h

```diff
@@ -456,4 +454,4 @@ class SaveFocussedXYETest : public CxxTest::TestSuite {
   const double m_tol;
 };
 
-#endif // SAVEFOCUSSEDXYETEST_H_
```

Comment:
I note that the new name is already used in the documentation and so that does not also need changing.